### PR TITLE
Feature/create page login

### PR DESCRIPTION
**Pull request by JRicardoT**
### What (if any) features are you implementing?
- Created a login form.
- Upon successful loggin the user sees their dashboard.

### What (if anything) did you refactor?
- Refactored the way I initialized my data.

### Were there any issues that arose?
- None.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -53,6 +53,19 @@ <h3>Your Trips</h3>
...
+      </div>
+      <div class="login-info-container">
+        <label for="password">Password:</label>
+        <input class="login-input" type="password" id="password" name="password" placeholder="Password" aria-required="true">
+      </div>
+      <button class="login-button" id="loginButton">LOGIN</button>
+      <p class="login-error-message hidden" id="loginErrorMessage">Please Fill Out All Sections!</p>
+    </form>
   </main>
   <!-- <h4 class="no-trips-message hidden" id="noTripsMessage">You Have No Pending Trips. Time To Book!</h4> -->
```
Comment (aburg15) [inline dist/index.html:70]: I may be missing something but you probably want to delete your commented out code on lines 70 - 72

File: src/css/base.scss [hunk 2]
```diff
@@ -159,4 +161,55 @@ h3 {
 
 .hidden {
   display: none;
+}
+
+// ~~~~~~~
+.login-form {
+  // border: 1px solid red;
```
Comment (aburg15) [inline src/css/base.scss:168]: Review your css file to make sure commented out code is removed.

File: src/css/base.scss [hunk 3]
```diff
@@ -159,4 +161,55 @@ h3 {
...
+  height: 50vh;
+  width: 55vh;
+  box-shadow: 0 10px 14px rgb(34 15 15 / 13%), 0 4px 14px rgb(0 0 0 / 22%);
+  // margin-top: 100px;
+  display: flex;
```
Comment (aburg15) [inline src/css/base.scss:173]: Opportunity to add a mixin for display flex, flex direction column, etc.  I also don't see any SASS variables currently in use. Lets discuss.

File: src/dom-updates.js [hunk 4]
```diff
@@ -45,7 +45,7 @@ let domUpdates = {
   },
```
Comment (aburg15) [inline src/dom-updates.js:46]: Nice job on ensuring the user enters the various inputs correctly!

File: dist/index.html [hunk 5]
```diff
@@ -53,6 +53,19 @@ <h3>Your Trips</h3>
     </div>
   </section>
   <main id="tripCardsContainer">
+    <h4 class="login-title" id="loginTitle">Login To Continue</h4>
+    <form class="login-form" id="loginForm">
+      <div class="login-info-container">
```
Comment (aburg15) [inline dist/index.html:58]: Nice use of aria and accessibility in general

File: dist/index.html [hunk 6]
```diff
@@ -53,6 +53,19 @@ <h3>Your Trips</h3>
...
+        <label for="userName">Username:</label>
+        <input class="login-input" type="text" id="userName" name="username" placeholder="Username" aria-required="true">
```
Comment (aburg15) [inline dist/index.html:60]: A thought for next time is to add 'required' to each input element.  The form will then not be able to be submitted without data in the input.

File: src/scripts.js [hunk 7]
```diff
@@ -23,34 +23,96 @@ let userInputForm = document.getElementById('userInputForm');
 let backButton = document.getElementById('backButton');
 let pendingTripsButton = document.getElementById('pendingTripsButton');
 let allTripsButton = document.getElementById('allTripsButton');
+// ~~~~~~~~~~~~login~~~~~~~~~~
```
Comment (aburg15) [inline src/scripts.js:26]: Do you need to include the section headers in your js file here (login, global variables, etc.).  Not a big deal but i thought i received a comment once that we should not include those.

**Review by Markus-Xavier — APPROVED** [review]
The way you are handling login validation is very neat way to do it! This all looks really good!

File: src/scripts.js [hunk 8]
```diff
@@ -23,34 +23,96 @@ let userInputForm = document.getElementById('userInputForm');
...
+let loginTitle = document.getElementById('loginTitle');
+let tripInputContainer = document.getElementById('tripInputContainer');
+let tripSelectorContainer = document.getElementById('tripSelectorContainer');
+
 // ~~~~~~~~~ global variables ~~~~~~~~~
 let allData;
 let currentTraveler;
 // ~~~~~~~~~ event listeners ~~~~~~~~~~
-window.addEventListener('load', displayData);
+window.addEventListener('load', getData);
```
Comment (aburg15) [inline src/scripts.js:40]: I think you can delete this

**Pull request merged**